data/avgi/avgi outfits.txt

```diff
@@ -0,0 +1,609 @@
+
+
+outfit "Optical Lasing Generator"
+	category "Systems"
+	cost 120000
+	thumbnail "outfit/unknown"
+	"mass" 30
+	"outfit space" -30
+	"lasing power" 3
+	description "A monolithic slab of semiconductor diode is used to efficiently pump a spool of doped optical fiber. Hundreds of kilometers long, the coiled fibers serve as a lasing medium with the power needed to produce an intense beam of light, and can be gain-switched to pulse at a variety of frequencies. The produced laser light, which can be converted to a number of wavelengths, can be easily tapped and routed via optical fibers or mirrors to a number of turrets on a ship, eliminating the need for the heavy lasing generator to be anywhere near the turret itself."
+	description "The infrared light produced in the coil can either be passed through a frequency doubling crystal to support up to three Green Optical Lasers, or frequency tripled to support a single Blue Optical Laser."
+
+
+
+# Power:
+
+
+outfit "Concentrating Solar Cell"
+	category "Power"
+	cost 12600
+	thumbnail "outfit/unknown"
+	"mass" 1
+	"outfit space" -3
+	"solar collection" 0.3
+	"solar heat" 0.12
+	description "This tiny set of concentrating solar cells efficiently collects sunlight, turning it into usable energy. Thin Fresnel lenses concentrate sunlight onto a small number of highly efficient solar cells, reducing weight and improving efficiency."
+
+
+outfit "Concentrating Solar Array"
+	category "Power"
+	cost 114000
+	thumbnail "outfit/unknown"
+	"mass" 3
+	"outfit space" -18
+	"solar collection" 2.4
+	"solar heat" 1.02
+	description "This large set of concentrating solar cells efficiently collects sunlight, turning it into usable energy. Thin Fresnel lenses concentrate sunlight onto a small number of highly efficient solar cells, reducing weight and improving efficiency."
+	description "While much more efficient than a single cell alone, the Solar Array also produces a disproportionate amount of heat."
+
+
+outfit "Photoelectric Lamina"
+	plural "Photoelectric Laminae"
+	category "Power"
+	cost 294000
+	thumbnail "outfit/photoelectric lamina"
+	"mass" 24
+	"anchor point" -1
+	"solar collection" 6
+	"solar heat" 1.8
+	description "These concentrating solar panels can be mounted conformally to a ship's hull, efficiently collecting sunlight and turning it into usable energy."
+	description "Photovoltaic Laminae take advantage of special external hardpoints built into most Avgi ships, allowing them to be mounted without taking up any outfit space. However, they also tend to increase the amount of heat absorbed from the local star."
+
+
+outfit "Lantern Fission Core"
```

Comment:
This is very strong for its size and tier, with 7.97 energy per net space (after cooling), compared to the comparable Dwarf Core which sits at 6.82
In the wider game balance view, this beats the 23 ton Candle's 7.12, and the Small Biochemical Cell, which works out at 7.68 once its energy capacity is accounted for. The only better outfits of a similar or smaller size are the ka'het MHD at 9.27 (22 outfit space), and the Coalition Large Collector Module at 9.13 (21 outfit space).
With its current stats the Lantern Fission Core is the best purchaseable non-solar generator at this size.
Not breaking anything as such, but probably still somewhat stronger than it should be?

Reply:
I think the 3.6 energy/frame that you've got it at now is about right. That puts it roughly on par with the Hai's Pebble, which is also a miniaturised fission reactor.